src/c++/tests/cc_client_test.cc

```diff
@@ -1300,6 +1300,42 @@ TYPED_TEST_P(ClientTest, LoadWithFileOverride)
         << "expect model " << override_name << " version " << vr.first
         << " readiness: " << vr.second;
   }
+
+  // Request to load the model with override config in original name
+  // Send the config with wrong format
+  config =
+      "\"parameters\": {\"config\": {{\"backend\":\"onnxruntime\", "
+      "\"version_policy\":{\"specific\":{\"versions\":[2]}}}}}";
+  err = this->LoadModel(model_name, config);
+  ASSERT_FALSE(err.IsOk()) << "Expect LoadModel() to fail";
+
+  // Send the config with the correct format
```

Comment:
May be good to add a check before this that the original model is unchanged.

Reply:
Also, this is beyond the scope of this ticket, but we might want to add "sections" to this test via comments or break it down into multiple tests. It's getting quite large.